src/components/metadataEditor/metadataEditor.js

```diff
@@ -638,6 +638,9 @@ import template from './metadataEditor.template.html';
 
         if (item.Type === 'Person') {
             //todo
```

Comment:
Todo what? :upside_down_face:

Reply:
Not sure what Luke wanted to do :yum: 
We can probably remove it though, yeah.